Makefile.ibi

```diff
@@ -24,10 +24,10 @@ OPENSHIFT_INSTALLER_BIN ?= $(OPENSHIFT_INSTALLER_DIR)/bin/openshift-install
 # Used to populate releaseRegistry in imagebased-config.yaml
 IBI_RELEASE_REGISTRY ?= quay.io
 
-IBI_INSTALL_ISO_TEMPLATE = image-based-install-iso-template.yaml
-IMAGEBASED_CONFIG_TEMPLATE = imagebased-config-template.yaml
+IBI_INSTALLATION_CONFIG_TEMPLATE = image-based-installation-config-template.yaml
+IBI_CONFIG_TEMPLATE = image-based-config-template.yaml
```

Comment:
Maybe we should also add a

```
IBI_RELEASE_IMAGE ?= $(DEFAULT_RELEASE_IMAGE)
```
and use that variable downer, instead of using the DEFAULT? 

The idea is that maybe we want to override IBI_RELEASE_IMAGE somehow, but not the DEFAULT one (for example when setting custom defaults via `.config-override` file